app/components/Machines.tsx

```diff
@@ -1,26 +1,52 @@
 import { Link as ChakraLink } from "@chakra-ui/next-js";
-import { Container, Heading, Box } from "@chakra-ui/react";
+import { Container, Heading, Box, Flex } from "@chakra-ui/react";
+import { PageQuery } from "@/tina/__generated__/types";
 
 import "slick-carousel/slick/slick-theme.css";
 import "slick-carousel/slick/slick.css";
 
 import MachineCarousel from "./MachineCarousel";
+import ContentWrapper from "./ContentWrapper";
+
+type Block = NonNullable<NonNullable<PageQuery["page"]["blocks"]>[number]>;
+
+export type MachineCardsBlock = Extract<
+  Block,
+  { __typename: "PageBlocksMachineCarousel" }
+>;
+
+export default function Machines({
+  width,
+  ...props
+}: { width: number } & MachineCardsBlock): JSX.Element {
+  const adjustedWidth = width <= 1728 ? width * 0.9 : 1728 * 0.9;
```

Comment:
Added logic for screens larger than 1728px. 

Btw, I found a small bug with using width that you get from `window.innerWidth`. It works great when the page loads for the first time; but if you try to resize the window, change in innerWidth is not picked up by MachineCarousel and you need to refresh the page to trigger a rerender. I didn't want to mess with this as Brett mentioned that he might revisit it later.

Reply:
That was the best solution I could come up with at the time. The React-slick library seems kind of limited in that regard. It has "responsive" in its settings API, but it doesn't resize reactively based on width, only for breakpoints. I wrapped the component in a Container and tried both width and maxWidth at 100% and lots of other attempts, but got unexpected results. The window.innerWidth was the only thing that worked at the time.

At the moment in this branch, I am seeing dark grey background bars on the side, so that somehow needs to be changed to white:
<img width="1462" alt="Screenshot 2024-02-17 at 12 23 56 PM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/76603041/810865eb-1cff-463c-865a-752db77718b2">

Reply:
Got it. I would say that using breakpoints might be a good idea (though I haven't tested how it works and looks) considering that we're relying on breakpoints for responsiveness anyway. Maybe though I'm missing something. 

As for the background, this is very strange. What screen size do you have when you see this background? As you can see on the screenshots that I've attached for this PR, there's no grey background. I wonder if there is some sort of difference in our set ups, because this is not the first time when things are displayed differently on your end. I'm talking mostly about Archaa's previous PRs when you also noted that the text colors are different from what they actually are. On my end, Archaa's components rendered with correct colors, so I'm not sure what's happening. 

@avinashi10 can you check if you also see grey background on this branch?

Reply:
I just tried it on Safari, Firefox and Opera and none of those had the grey bars, so it is definitely something to do with my Chrome setup that I need to look into, and not your code.

Reply:
*Update: I reset all of my Chrome settings to defaults and that fixed it. Now everything looks fine. Sorry for the confusion :)